### PR TITLE
Add manifest

### PR DESCRIPTION
**Pull request by slackmon**
###  Summary

Add a manifest and instructions on how to use the manifest to easily create a Bolt App

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

**Review by seratch — APPROVED** [review]
LGTM 👍 

To: other maintainers
Feel free to merge this PR during NA business hours tomorrow

File: README.md [hunk 1]
```diff
@@ -8,12 +8,23 @@ responding to events and interactive buttons.
 
 ## Running locally
 
+### 0. Create a new Slack App
+
+- Go to https://api.slack.com/apps
+- Click **Create App**
+- Choose a workspace
+- Enter App Manifest using contents of `manifest.yaml`
```
Comment (misscoded) [inline README.md:16]: I know we opted for JSON in the template apps we created that make use of the manifest. Do we want to continue that trend for this? cc: @mwbrooks @stevengill

Comment (filmaj) [reply README.md:16]: My intuition is that bolt-js users will probably prefer JSON over YAML but that's just my speculation.

**Review by filmaj — APPROVED** [review]
LGTM though @misscoded brings up a good question re: JSON vs. YAML manifest format and what we should standardize on.

**Pull request merged**